chcemvediet/apps/inforequests/sitemaps.py

```diff
@@ -26,3 +29,16 @@ def lastmod(self, (lang, inforequest)):
         return (Action.objects
                 .of_inforequest(inforequest)
                 .aggregate(most_recent=Max(u'created'))[u'most_recent'])
+
+class PublishedInforequestsSitemap(Sitemap):
```

Comment:
Nepride ti nazov `PublishedInforequestsSitemap` zmatocny? Nazov cohokolvek sluzi na to, aby z nazvu bodlo jasne, co to je, a aby z nazvoch dvoch veci sa dalo odlisit, v com je medzi nimi rozdiel. Naozaj ti pride OK nazvat sitemap pre detaily zverejnenych infoziadosti `InforequestsSitemap` a pre ich zoznam `PublishedInforequestsSitemap`?

Zmen im nazvy tak, aby z nazvov bolo ocividne, v com je medzi nimi rozdiel.